### DISCUSSION TITLE
Puzzle in Chapter 7, Q7

### DISCUSSION THREAD

**Discussion by YinhaoHu** [xxyzz/ostep-hw#18] [General]
For the wors-case, the quantum length might be longer  than every job's length, in this case, the average response time might be 
Sum(job[i].turnaround-time - job[0].turnaround-time)/n (i: 0~n-1), which would be approve by 
`python ./scheduler.py -p RR -q 100 -l 50,50 -c`
How would you like that?

**Comment by xxyzz** [comment]
Thanks! I update the answer in https://github.com/xxyzz/ostep-hw/commit/b2614bf35ff22fbd501c756eb5cd2f339423b295.
